app/frontend/js/datetimepicker.js

```diff
@@ -99,31 +83,7 @@ function hasUserChangedDate(oldDate, newDate) {
 }
 
 function registerFocusHandlers(datetimePicker, element) {
-  // Show datetimepicker when user clicks in text field next to button
-  // or when input field receives focus
-  let isButtonInvokingFocus = false;
-
   element.find(".td-input").on("click focusin", (_e) => {
-    try {
-      if (!isButtonInvokingFocus) {
-        datetimePicker.show();
-      }
-    }
-    finally {
-      isButtonInvokingFocus = false;
-    }
-  });
-
-  element.find(".td-picker-button").on("click", () => {
-    isButtonInvokingFocus = true;
-    element.find(".td-input").focus();
-  });
-
-  // Hide datetimepicker when input field loses focus
-  element.find(".td-input").blur((e) => {
-    if (!e.relatedTarget) {
-      return;
-    }
-    datetimePicker.hide();
+    datetimePicker.show();
   });
 }
```

Comment:
It seems that this is unstable under "Back/Forward" button in the browser (possibly a turbo problem), I uploaded a video into basecamp as upload fails here due to its size.